integrations/jest-plugin/src/index.ts

```diff
@@ -21,10 +25,10 @@ export function process(src: string, filename: string): string {
     });
     const mapBase64 = Buffer.from(JSON.stringify(sourceMap)).toString("base64");
     const suffix = `//# sourceMappingURL=data:application/json;charset=utf-8;base64,${mapBase64}`;
-    return `${code}\n${suffix}`;
+    return {code: `${code}\n${suffix}`, map: sourceMap};
```

Comment:
Now that we're returning a source map, I bet we can remove `suffix` and get rid of the previous two lines. Let me try it to confirm; from https://github.com/alangpierce/sucrase/pull/661 , my way of testing originally was to try it out in WebStorm and see if breakpoints work.

Reply:
Unfortunately, from some testing, it looks like the `sourceMappingURL` suffix is still necessary for breakpoints to work right in WebStorm, so it looks like it makes sense to keep.